charts/backstage/Chart.yaml

```
@@ -47,4 +47,4 @@ sources: []
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 # Note that when this chart is published to https://github.com/openshift-helm-charts/charts
 # it will follow the RHDH versioning 1.y.z
-version: 4.4.0
+version: 4.5.0
```

```suggestion
version: 4.4.1
```

This is for a bug fix.

---

ah yes, thanks